diff --git a/collections/luci/Makefile b/collections/luci/Makefile
index 399f0a1..4041434 100644
--- a/collections/luci/Makefile
+++ b/collections/luci/Makefile
@@ -13,8 +13,7 @@ LUCI_TITLE:=LuCI interface with Uhttpd as Webserver (default)
 LUCI_DESCRIPTION:=Standard OpenWrt set including package management and attended sysupgrades support
 LUCI_DEPENDS:= \
 	+luci-light \
-	+luci-app-package-manager \
-	+luci-app-attendedsysupgrade
+	+luci-app-package-manager
 
 PKG_LICENSE:=Apache-2.0
 
